docs/using.rst

```diff
@@ -860,6 +891,12 @@ in the config file.
 Log Rotation
 ============
 
+.. note:: Some distributions, including Debian and Ubuntu, disable
+   certbot's internal log rotation in favor of a more traditional
+   logrotate script.  If you are using a distribution's packages and
+   want to alter the log rotation, check `/etc/logrotate.d/` for a
+   certbot rotation script.
+
```

Comment:
Similarly I'd move this to be after the initial paragraph in this section. I think it makes more sense to explain the default log behavior and then have a note about a special case.